app/views/ops/_all_tabs.html.haml

```diff
@@ -125,8 +123,6 @@
           = render :partial => "diagnostics_servers_roles_tab"
         = miq_tab_content("diagnostics_server_list", @sb[:active_tab]) do
           = render :partial => "diagnostics_server_list_tab"
-        = miq_tab_content("diagnostics_collect_logs", @sb[:active_tab]) do
-          = render :partial => "diagnostics_collect_logs_tab"
```

Comment:
With this partial removed here and below you can also delete `app/views/ops/_diagnostics_collect_logs_tab.html.haml`